cmd/configure.go

```diff
@@ -3,89 +3,110 @@ package cmd
 import (
 	"errors"
 	"fmt"
+	"strings"
 
 	"github.com/PossibleLlama/worklog/model"
 
 	"github.com/spf13/cobra"
 )
 
-const defaultAuthor = ""
-const defaultDuration = 15
+const (
+	configDefaultAuthor   = ""
+	configDefaultDuration = 15
+	configDefaultFormat   = "pretty"
+)
 
-var providedAuthor string
-var providedDuration int
-var providedFormat string
+var configProvidedAuthor string
+var configProvidedDuration int
+var configProvidedFormat string
 
 // configureCmd represents the create command
 var configureCmd = &cobra.Command{
 	Use:   "configure",
 	Short: "Setup configuration for the application",
 	Long: `Setup configuration file for the application,
 setting up the defaults and adding in passed arguments.`,
-	Args: func(cmd *cobra.Command, args []string) error {
-		providedAuthor = defaultAuthor
-		providedDuration = defaultDuration
-		return nil
-	},
-	RunE: func(cmd *cobra.Command, args []string) error {
-		return callService()
-	},
+	Args: ConfigArgs,
+	RunE: ConfigRun,
+}
+
+// ConfigArgs public method to validate arguments
+func ConfigArgs(cmd *cobra.Command, args []string) error {
+	return configArgs()
+}
+
+func configArgs() error {
+	configProvidedAuthor = configDefaultAuthor
+	configProvidedDuration = configDefaultDuration
+	configProvidedFormat = configDefaultFormat
+	return nil
+}
+
+// ConfigRun public method to run configuration
+func ConfigRun(cmd *cobra.Command, args []string) error {
+	return configRun()
+}
+
+func configRun() error {
+	cfg := model.NewConfig(configProvidedAuthor, configProvidedFormat, configProvidedDuration)
+	if err := wlService.Configure(cfg); err != nil {
+		return err
+	}
+	fmt.Println("Successfully configured")
+	return nil
 }
 
-var defaultsCmd = &cobra.Command{
+var overrideDefaultsCmd = &cobra.Command{
 	Use:   "defaults",
```

Comment:
While changing other details, this should probably change to `overrideDefaults` as well.